medicines/pars-upload/src/products_form.js

```diff
@@ -73,8 +73,8 @@ export const Products = ({
       <form onSubmit={onSubmit} ref={formRef}>
         <FormGroup>
           <Field
-            name="product_name"
```

Comment:
If the field name is changed this will need to be updated on the backend too

Reply:
reverted back to `product_name` :+1: